Functions/xASL_quant_FEAST.m

```diff
@@ -54,15 +54,17 @@ function xASL_quant_FEAST(x)
     % Load data
     CBF{iSession} = xASL_io_Nifti2Im(PathCBF{iSession});
     Gradient{iSession} = xASL_io_Nifti2Im(fullfile(x.D.PopDir, ['SliceGradient_extrapolated_' x.P.SubjectID '_' x.SESSIONS{iSession} '.nii']));
-    % correct different PLD scales
-	if length(x.Q.SliceReadoutTime) == 1
-		PLD{iSession} = x.Q.Initial_PLD + ((Gradient{iSession}-1) .* x.Q.SliceReadoutTime);
-	else
-		imGradientRounded = round(Gradient{iSession});
-		imGradientRounded(imGradientRounded<1) = 1;
-		imGradientRounded(imGradientRounded>length(x.Q.SliceReadoutTime)) = length(x.Q.SliceReadoutTime);
-		PLD{iSession} = x.Q.Initial_PLD + x.Q.SliceReadoutTime(imGradientRounded);
-	end
+	% Obtain the correct SliceTime
+	imASL = xASL_io_ReadNifti(x.P.Path_ASL4D);
+	nSlices = size(imASL.dat,3);
+	SliceTime = xASL_quant_SliceTimeVector(x,nSlices);
+	
+    % Correct different PLD scales
+	imGradientRounded = round(Gradient{iSession});
```

Comment:
Perhaps we should rename `Gradient` to `SliceNumber` or something more self-explanatory

Reply:
Yes! It was a bit ambiguous if this was in standard or native space etc. So SliceNumber would be totally helpful. Shall I do it now?

Reply:
It's loaded from NII called 'SliceGradient_extrapolated'. Should I still rename?!

Reply:
Yes, SliceNumber would work. Native and standard spaces are recognized by their folder (subject or population).
There is indeed a separate extrapolated SliceGradient, which avoids the NaNs outside the original FoV when resampling. This one is useful for the quantification, while the one without the extrapolation is useful for QC (displaying the native spaces slices in standard space).

Reply:
OK. I will change the name in this file and then merge.

Reply:
I have renamed it both for quant_FEAST and quant_SinglePLD -> gradient to slice-number.

In SinglePLD, I have avoided creating a new matrix for the rounded version of SliceNumber -> it can replace the old one.
in FEAST, I have done the same and also simplified certain things not to keep unnecessary copies of certain variables.